numbat-cli/tests/integration.rs

```diff
@@ -33,8 +33,8 @@ fn pass_expression_on_command_line() {
         .arg("--expression")
         .arg("2 ++ 3")
         .assert()
-        .failure()
-        .stderr(predicates::str::contains("while parsing"));
+        .success()
+        .stdout(predicates::str::contains("5"));
```

Comment:
This test was here to make sure that we fail with a non-zero exit status on parser errors. I suggest we revert this and change the expression instead (e.g. `2 /+ 3`.

Reply:
Yeah, except, hehe, now `2 / +3` is a valid expression 😎 

I updated it to `2 +/ 3`

Reply:
Oh. I should have invested more than 2 seconds of thought into that example :smile: